catalystwan/endpoints/configuration/feature_profile/sdwan/policy_object.py

```diff
@@ -0,0 +1,87 @@
+# mypy: disable-error-code="empty-body"
+# flake8: noqa
```

Comment:
please don't forget to remove that before merging to main.

Reply:
will delete